chart/develop/values.yaml

```diff
@@ -15,3 +15,4 @@ etcd:
   persistence:
     enabled: false
 
+mayastorLogLevel: debug
```

Comment:
I think we want "mayastor=debug" otherwise we'll get debug from all crates/libraries we make use of?

Reply:
`mayastor=` is directly in the template

https://github.com/openebs/Mayastor/pull/927/files#diff-35937fcb9923c47951a5557076cfeabc39258b78547d11f878e82453dccbbcd7R38

Reply:
hmm sorry! I think what I said is slightly wrong, we'd want "**info,**mayastor=debug", otherwise we'd actually lose info level messages from the crates/libraries we make use of, including the local ones from our repo. So that basically means, info for all, and for the mayastor crate, use debug.
Example, this one:  WARN **mbus_api**::mbus_nats:mbus_nats.rs:77] Connection to the NATS server has been lost.
https://rust-lang-nursery.github.io/rust-cookbook/development_tools/debugging/config_log.html

Reply:
Oh I see! Thanks for clarification. Let me fix that.